cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -3390,50 +3381,66 @@ pub(crate) fn define(
         .operands_out(vec![x]),
     );
 
-    let FloatScalar = &TypeVar::new(
-        "FloatScalar",
-        "A scalar only floating point number",
-        TypeSetBuilder::new().floats(Interval::All).build(),
-    );
-    let x = &Operand::new("x", FloatScalar);
-    let a = &Operand::new("a", IntTo);
+    {
```

Comment:
I strongly sympathize with wanting to limit the scope of definitions, but introducing a block here is contrary to the prevailing style and I'm not convinced it makes this easier to understand. I would like to figure out how to fix the prevailing style so that it _is_ easier to understand, but I'd rather not do that in this PR. Would you drop the braces here?